### PR TITLE
Geocoding field for hi res batching

### PR DESCRIPTION
**Pull request by rafatower**
@Kartones can you please review?

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3741/))

File: db/migrate/20150518151555_add_geocoding_batching.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# encoding: utf-8
+
+class AddBatchedToGeocoding < Sequel::Migration
+
+  def up
+    alter_table :geocodings do
+      add_column :batched, :boolean
```
Comment (Kartones) [inline db/migrate/20150518151555_add_geocoding_batching.rb]: No default value...

Comment (rafatower) [inline db/migrate/20150518151555_add_geocoding_batching.rb]: on purpose, otherwise I'll set NULL (the default) and I think also for internal geocoder

Comment (Kartones) [inline db/migrate/20150518151555_add_geocoding_batching.rb]: ok!

**Comment by Kartones** [comment]
:+1:

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3748/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3752/))

**Pull request merged**